addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java

```diff
@@ -164,8 +164,7 @@ public boolean servicePing(String host, int port, int timeout) throws IOExceptio
         } catch (SocketTimeoutException ignored) {
             return false;
         } catch (ConnectException e) {
-            // Connection refused, there is a device on the other end though
-            return true;
+            return false;
```

Comment:
Since all these exceptions are now handled similarly you can simplify the code with a multi catch: 

```
} catch (ConnectException | NoRouteToHostException | SocketTimeoutException e) {
    return false;
}
```